packages/components/organisms/f-checkout/src/components/Selector.vue

```diff
@@ -132,17 +132,17 @@ export default {
          */
         initFulfilmentTime () {
             const times = this.fulfilmentTimes || [];
+            const foundTimeInList = (times.some(i => i.value === this.time.from) && this.time.from);
```

Comment:
Can we try to find a better name for this? I know I wrote this, but it was while we were trying to fix the issue quickly 😬

Reply:
Is `isTimeAvailable` ok @SachaZvetelman ?

Reply:
Changed to `preSelectedTime`